diagrams/request_uri_mode_create.md

```diff
@@ -0,0 +1,49 @@
+```plantuml
+@startuml
+
+autonumber
+
+box "Wallet"
+participant "Metadata" as wm
+participant "Authorization Endpoint" as w
+end box
+
+participant "User Agent" as u
+
+box "Verifier"
+participant "Frontend" as r
+participant "Request Endpoint" as rp
+participant "Response Endpoint" as rb
+end box
+
+u --> r : use
+activate r
+
+r --> u: **signed authorization request**\n(client_id, request_uri, request_uri_mode=create, state)
+deactivate r
+u --> w: **signed authorization request**\n(client_id, request_uri, request_uri_mode=create, state)
```

Comment:
4. (Can be ignored if signing gets removed, see 1.) What object is used to convey the signed Authz Request in this step? Is it `request` as defined in JAR? NOTE: For OAuth2 backwards compatibility, `client_id` has to be provided always as a query string parameter (outside of the Request Object) as well.

Reply:
good question. I think we should use `request` with the content like described in the diagram being request_uri, request_uri_mode (and optionally client_id_scheme).

Reply:
modified diagram and text to send the signed object as `request`